### DISCUSSION TITLE
ARM64 support for the KEDA operator

### DISCUSSION THREAD

**Discussion by odidev** [kedacore/keda#4741] [General] [answered]
Hi , 

I have been trying to [install](https://keda.sh/docs/2.10/deploy/) `KEDA` from the `OperatorHub` on the **Arm64** platform.   

After deploying an OpenShift cluster on Arm64 when I search for KEDA operator in the operator hub, it shows `"no results found"`.  
  
![Screenshot (554)](https://github.com/kedacore/keda/assets/40816837/ab7b857f-85a7-4d2f-917a-05c63dc0b1f6)

Do you have any plans to release it for Arm64?

**Comment by tomkerkhove** [comment]
@zroubalik This is probably for the OLM operator or am I wrong?

**Reply by zroubalik** [reply, marked as answer]
@odidev could you please open a support ticket on the Red Hat side?
